portal/server/utils/csv-utils.js

```diff
@@ -41,6 +41,28 @@ const excelColumnToColumnIndex = (column) => {
   return result - 1;
 };
 
+/**
+ * Checks for floating-point rounding errors and rounds to two decimal places if within error tolerance.
+ * @param {number} number - The number to check and potentially round.
+ * @returns {number} - The rounded number to two decimal places if within tolerance, otherwise the original number.
+ * @throws {TypeError} - If the input is not a number.
+ */
+const handleFloatingPointRoundingErrors = (number) => {
+  if (typeof number !== 'number') {
+    throw new TypeError('Input must be a number');
+  }
+
+  const TOLERANCE = 1e-6;
```

Comment:
Can these be added to `libs/common` to future proof these very specific constants.

Reply:
Pulled these out into `libs/common` thanks